{
	"document": {
		"aggregate_severity": {
			"namespace": "https://access.redhat.com/security/updates/classification/",
			"text": "Moderate"
		},
		"category": "csaf_vex",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright Â© Red Hat, Inc. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "An update for python-django is now available for Red Hat OpenStack Platform\n16.1 (Train).\n\nRed Hat Product Security has rated this update as having a security impact\nof Moderate. A Common Vulnerability Scoring System (CVSS) base score, which\ngives a detailed severity rating, is available for each vulnerability from\nthe CVE link(s) in the References section.",
				"title": "Topic"
			},
			{
				"category": "general",
				"text": "Django is a high-level Python Web framework that encourages rapid\ndevelopment and a clean, pragmatic design. It focuses on automating as much\nas possible and adhering to the DRY (Don't Repeat Yourself) principle.\n\nSecurity Fix(es):\n\n* potential data leakage via malformed memcached keys (CVE-2020-13254)\n\nFor more details about the security issue(s), including the impact, a CVSS\nscore, acknowledgments, and other related information, refer to the CVE\npage listed in the References section.",
				"title": "Details"
			},
			{
				"category": "legal_disclaimer",
				"text": "This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original.",
				"title": "Terms of Use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://access.redhat.com/security/team/contact/",
			"issuing_authority": "Red Hat Product Security is responsible for vulnerability handling across all Red Hat offerings.",
			"name": "Red Hat Product Security",
			"namespace": "https://www.redhat.com"
		},
		"references": [
			{
				"category": "self",
				"summary": "https://access.redhat.com/errata/RHSA-2021:0915",
				"url": "https://access.redhat.com/errata/RHSA-2021:0915"
			},
			{
				"category": "external",
				"summary": "https://access.redhat.com/security/updates/classification/#moderate",
				"url": "https://access.redhat.com/security/updates/classification/#moderate"
			},
			{
				"category": "external",
				"summary": "1843614",
				"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1843614"
			},
			{
				"category": "external",
				"summary": "1919306",
				"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1919306"
			},
			{
				"category": "self",
				"summary": "Canonical URL",
				"url": "https://access.redhat.com/security/data/csaf/v2/advisories/2021/rhsa-2021_0915.json"
			}
		],
		"title": "Red Hat Security Advisory: Red Hat OpenStack Platform 16.1.4 (python-django) security update",
		"tracking": {
			"current_release_date": "2024-04-21T09:05:18+00:00",
			"generator": {
				"date": "2024-04-21T09:05:18+00:00",
				"engine": {
					"name": "Red Hat SDEngine",
					"version": "3.29.1"
				}
			},
			"id": "RHSA-2021:0915",
			"initial_release_date": "2021-03-17T15:07:57+00:00",
			"revision_history": [
				{
					"date": "2021-03-17T15:07:57+00:00",
					"number": "1",
					"summary": "Initial version"
				},
				{
					"date": "2021-03-17T15:07:57+00:00",
					"number": "2",
					"summary": "Last updated version"
				},
				{
					"date": "2024-04-21T09:05:18+00:00",
					"number": "3",
					"summary": "Last generated version"
				}
			],
			"status": "final",
			"version": "3"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_name",
								"name": "Red Hat OpenStack Platform 16.1",
								"product": {
									"name": "Red Hat OpenStack Platform 16.1",
									"product_id": "8Base-RHOS-16.1",
									"product_identification_helper": {
										"cpe": "cpe:/a:redhat:openstack:16.1::el8"
									}
								}
							}
						],
						"category": "product_family",
						"name": "Red Hat OpenStack Platform"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python-django20-0:2.0.13-12.el8ost.src",
								"product": {
									"name": "python-django20-0:2.0.13-12.el8ost.src",
									"product_id": "python-django20-0:2.0.13-12.el8ost.src",
									"product_identification_helper": {
										"purl": "pkg:rpm/redhat/python-django20@2.0.13-12.el8ost?arch=src"
									}
								}
							}
						],
						"category": "architecture",
						"name": "src"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python-django20-bash-completion-0:2.0.13-12.el8ost.noarch",
								"product": {
									"name": "python-django20-bash-completion-0:2.0.13-12.el8ost.noarch",
									"product_id": "python-django20-bash-completion-0:2.0.13-12.el8ost.noarch",
									"product_identification_helper": {
										"purl": "pkg:rpm/redhat/python-django20-bash-completion@2.0.13-12.el8ost?arch=noarch"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python3-django20-0:2.0.13-12.el8ost.noarch",
								"product": {
									"name": "python3-django20-0:2.0.13-12.el8ost.noarch",
									"product_id": "python3-django20-0:2.0.13-12.el8ost.noarch",
									"product_identification_helper": {
										"purl": "pkg:rpm/redhat/python3-django20@2.0.13-12.el8ost?arch=noarch"
									}
								}
							}
						],
						"category": "architecture",
						"name": "noarch"
					}
				],
				"category": "vendor",
				"name": "Red Hat"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-django20-0:2.0.13-12.el8ost.src as a component of Red Hat OpenStack Platform 16.1",
					"product_id": "8Base-RHOS-16.1:python-django20-0:2.0.13-12.el8ost.src"
				},
				"product_reference": "python-django20-0:2.0.13-12.el8ost.src",
				"relates_to_product_reference": "8Base-RHOS-16.1"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-django20-bash-completion-0:2.0.13-12.el8ost.noarch as a component of Red Hat OpenStack Platform 16.1",
					"product_id": "8Base-RHOS-16.1:python-django20-bash-completion-0:2.0.13-12.el8ost.noarch"
				},
				"product_reference": "python-django20-bash-completion-0:2.0.13-12.el8ost.noarch",
				"relates_to_product_reference": "8Base-RHOS-16.1"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python3-django20-0:2.0.13-12.el8ost.noarch as a component of Red Hat OpenStack Platform 16.1",
					"product_id": "8Base-RHOS-16.1:python3-django20-0:2.0.13-12.el8ost.noarch"
				},
				"product_reference": "python3-django20-0:2.0.13-12.el8ost.noarch",
				"relates_to_product_reference": "8Base-RHOS-16.1"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2020-13254",
			"cwe": {
				"id": "CWE-200",
				"name": "Exposure of Sensitive Information to an Unauthorized Actor"
			},
			"discovery_date": "2020-05-27T00:00:00+00:00",
			"ids": [
				{
					"system_name": "Red Hat Bugzilla ID",
					"text": "1843614"
				}
			],
			"notes": [
				{
					"category": "description",
					"text": "A flaw was found in Django, where the memcached backend does not perform key validation and passes malformed keys. This flaw causes a key collision and potential data leakage. The highest threat from this vulnerability is to confidentiality.",
					"title": "Vulnerability description"
				},
				{
					"category": "summary",
					"text": "django: potential data leakage via malformed memcached keys",
					"title": "Vulnerability summary"
				},
				{
					"category": "other",
					"text": "Red Hat Satellite 6 ships affected python-django, however, it does not use memcached implementation in product code hence not vulnerable to this flaw.\n\nRed Hat Update Infrastructure 3 ships an affected version of python-django, however it does not use memcached as a cache backend and it is not vulnerable to this flaw.\n\nRed Hat Ceph Storage(RHCS) ships an affected version of python-django used with calamari and graphite which are no longer supported, hence the django package will not be fixed for RHCS.",
					"title": "Statement"
				},
				{
					"category": "general",
					"text": "The CVSS score(s) listed for this vulnerability do not reflect the associated product's status, and are included for informational purposes to better understand the severity of this vulnerability.",
					"title": "CVSS score applicability"
				}
			],
			"product_status": {
				"fixed": [
					"8Base-RHOS-16.1:python-django20-0:2.0.13-12.el8ost.src",
					"8Base-RHOS-16.1:python-django20-bash-completion-0:2.0.13-12.el8ost.noarch",
					"8Base-RHOS-16.1:python3-django20-0:2.0.13-12.el8ost.noarch"
				]
			},
			"references": [
				{
					"category": "self",
					"summary": "Canonical URL",
					"url": "https://access.redhat.com/security/cve/CVE-2020-13254"
				},
				{
					"category": "external",
					"summary": "RHBZ#1843614",
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1843614"
				},
				{
					"category": "external",
					"summary": "https://www.cve.org/CVERecord?id=CVE-2020-13254",
					"url": "https://www.cve.org/CVERecord?id=CVE-2020-13254"
				},
				{
					"category": "external",
					"summary": "https://nvd.nist.gov/vuln/detail/CVE-2020-13254",
					"url": "https://nvd.nist.gov/vuln/detail/CVE-2020-13254"
				},
				{
					"category": "external",
					"summary": "https://www.djangoproject.com/weblog/2020/jun/03/security-releases",
					"url": "https://www.djangoproject.com/weblog/2020/jun/03/security-releases"
				}
			],
			"release_date": "2020-06-03T00:00:00+00:00",
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "For details on how to apply this update, which includes the changes\ndescribed in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258",
					"product_ids": [
						"8Base-RHOS-16.1:python-django20-0:2.0.13-12.el8ost.src",
						"8Base-RHOS-16.1:python-django20-bash-completion-0:2.0.13-12.el8ost.noarch",
						"8Base-RHOS-16.1:python3-django20-0:2.0.13-12.el8ost.noarch"
					],
					"restart_required": {
						"category": "none"
					},
					"url": "https://access.redhat.com/errata/RHSA-2021:0915"
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"attackComplexity": "HIGH",
						"attackVector": "NETWORK",
						"availabilityImpact": "NONE",
						"baseScore": 5.9,
						"baseSeverity": "MEDIUM",
						"confidentialityImpact": "HIGH",
						"integrityImpact": "NONE",
						"privilegesRequired": "NONE",
						"scope": "UNCHANGED",
						"userInteraction": "NONE",
						"vectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:N/A:N",
						"version": "3.1"
					},
					"products": [
						"8Base-RHOS-16.1:python-django20-0:2.0.13-12.el8ost.src",
						"8Base-RHOS-16.1:python-django20-bash-completion-0:2.0.13-12.el8ost.noarch",
						"8Base-RHOS-16.1:python3-django20-0:2.0.13-12.el8ost.noarch"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"details": "Moderate"
				}
			],
			"title": "django: potential data leakage via malformed memcached keys"
		}
	]
}
